ledger/participant-integration-api/src/main/resources/db/migration/h2database/V102_100__rename_packages_size_column.sql

```diff
@@ -0,0 +1,5 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+ALTER TABLE packages
```

Comment:
we have currently h2database and h2database-appendonly directories, this should be going to the appnedonly, righ? (because we deal hear with the appendonly changes I assume)

moreover: perheaps you noticed that the h2database-appendonly is starting from scratch, because we do not need data continuity there. therefore It would be much easier just to change the V1__Append_only_schema.sql directly, and regenerate the hash

Reply:
if you want to have clarity about how migrations play together with append-only: FlywayMigrations

Reply:
reverted